src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -398,12 +409,16 @@ private void InitNativeProcess()
             // redirecting anything. This is a bit tricky as we always run redirected so
             // we have to see if the redirection is actually being done at the topmost level or not.
 
-            //Calculate if input and output are redirected.
-            bool redirectOutput;
-            bool redirectError;
-            bool redirectInput;
+            // Calculate if input and output are redirected.
+            bool redirectOutput = true;
+            bool redirectError = true;
+            bool redirectInput = true;
 
-            CalculateIORedirection(out redirectOutput, out redirectError, out redirectInput);
+            // If we are transcribing and can't scrape, then continue to redirect so we have the output
+            if (!this.Command.Context.EngineHostInterface.UI.IsTranscribing || _scrapeHostOutput)
```

Comment:
Suggestion: `this.Command.Context.EngineHostInterface.UI.IsTranscribing` is used at 3 places in this class. Maybe we can have an instance field to cache the value? Like `_isTranscribing = this.Command.Context.EngineHostInterface.UI.IsTranscribing` in the constructor?